auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -632,7 +633,15 @@ public FlowParameters getFlowParams() {
                                       mTheme,
                                       mLogo,
                                       mTosUrl,
-                                      mIsSmartLockEnabled);
+                                      mIsSmartLockEnabled,
+                                      mCreateAccountIfEmailNotExists);
         }
+
+        public SignInIntentBuilder setCreateAccountIfEmailNotExists(boolean enabled) {
```

Comment:
I think we can pick a shorter and/or clearer name for this.  Suggestions:

- `setAllowNewEmailUsers()`
- `setAllowNewEmailAccounts()`
- `setEnableEmailAccountCreation()`
- `setDisableEmailAccountCreation()` --> in this case we'd have to invert the boolean throughout